apps/expo-go/ios/Podfile.lock

```diff
@@ -414,17 +414,17 @@ PODS:
   - GoogleUtilities/UserDefaults (8.0.2):
     - GoogleUtilities/Logger
     - GoogleUtilities/Privacy
-  - hermes-engine (0.76.3):
-    - hermes-engine/cdp (= 0.76.3)
-    - hermes-engine/Hermes (= 0.76.3)
-    - hermes-engine/inspector (= 0.76.3)
-    - hermes-engine/inspector_chrome (= 0.76.3)
-    - hermes-engine/Public (= 0.76.3)
-  - hermes-engine/cdp (0.76.3)
-  - hermes-engine/Hermes (0.76.3)
-  - hermes-engine/inspector (0.76.3)
-  - hermes-engine/inspector_chrome (0.76.3)
-  - hermes-engine/Public (0.76.3)
+  - hermes-engine (0.77.0-rc.6):
```

Comment:
do we have 0.77 for expo-go already? would be good to double check before merge

Reply:
That's due to an issue in https://github.com/facebook/react-native/blob/main/packages/react-native/sdks/hermes-engine/hermes-engine.podspec#L10-L15 where `REACT_NATIVE_PATH` is being ignored. I'll open a PR on core fixing it

Reply:
here it is https://github.com/facebook/react-native/pull/48574